src/main/java/ai/timefold/solver/benchmarks/competitive/flowshop/FlowShopDataset.java

```diff
@@ -0,0 +1,661 @@
+    Ta064("taillard93", "Ta064.txt", 100, 5014),
+    Ta065("taillard93", "Ta065.txt", 100, 5250),
+    Ta066("taillard93", "Ta066.txt", 100, 5135),
+    Ta067("taillard93", "Ta067.txt", 100, 5246),
+    Ta068("taillard93", "Ta068.txt", 100, 5094),
+    Ta069("taillard93", "Ta069.txt", 100, 5448),
+    Ta070("taillard93", "Ta070.txt", 100, 5322),
+    Ta071("taillard93", "Ta071.txt", 100, 5770),
+    Ta072("taillard93", "Ta072.txt", 100, 5349),
+    Ta073("taillard93", "Ta073.txt", 100, 5676),
+    Ta074("taillard93", "Ta074.txt", 100, 5781),
+    Ta075("taillard93", "Ta075.txt", 100, 5467),
+    Ta076("taillard93", "Ta076.txt", 100, 5303),
+    Ta077("taillard93", "Ta077.txt", 100, 5595),
+    Ta078("taillard93", "Ta078.txt", 100, 5617),
+    Ta079("taillard93", "Ta079.txt", 100, 5871),
+    Ta080("taillard93", "Ta080.txt", 100, 5845),
+    Ta081("taillard93", "Ta081.txt", 100, 6134),
+    Ta082("taillard93", "Ta082.txt", 100, 6183),
+    Ta083("taillard93", "Ta083.txt", 100, 6252),
+    Ta084("taillard93", "Ta084.txt", 100, 6254),
+    Ta085("taillard93", "Ta085.txt", 100, 6270),
+    Ta086("taillard93", "Ta086.txt", 100, 6311),
+    Ta087("taillard93", "Ta087.txt", 100, 6223),
+    Ta088("taillard93", "Ta088.txt", 100, 6367),
+    Ta089("taillard93", "Ta089.txt", 100, 6246),
+    Ta090("taillard93", "Ta090.txt", 100, 6404),
+    Ta091("taillard93", "Ta091.txt", 200, 10862),
+    Ta092("taillard93", "Ta092.txt", 200, 10480),
+    Ta093("taillard93", "Ta093.txt", 200, 10922),
+    Ta094("taillard93", "Ta094.txt", 200, 10889),
+    Ta095("taillard93", "Ta095.txt", 200, 10524),
+    Ta096("taillard93", "Ta096.txt", 200, 10329),
+    Ta097("taillard93", "Ta097.txt", 200, 10854),
+    Ta098("taillard93", "Ta098.txt", 200, 10730),
+    Ta099("taillard93", "Ta099.txt", 200, 10438),
+    Ta100("taillard93", "Ta100.txt", 200, 10675),
+    Ta101("taillard93", "Ta101.txt", 200, 11158),
+    Ta102("taillard93", "Ta102.txt", 200, 11160),
+    Ta103("taillard93", "Ta103.txt", 200, 11281),
+    Ta104("taillard93", "Ta104.txt", 200, 11275),
+    Ta105("taillard93", "Ta105.txt", 200, 11259),
+    Ta106("taillard93", "Ta106.txt", 200, 11176),
+    Ta107("taillard93", "Ta107.txt", 200, 11337),
+    Ta108("taillard93", "Ta108.txt", 200, 11301),
+    Ta109("taillard93", "Ta109.txt", 200, 11146),
+    Ta110("taillard93", "Ta110.txt", 200, 11284),
+    Ta111("taillard93", "Ta111.txt", 500, 26040),
+    Ta112("taillard93", "Ta112.txt", 500, 26500),
+    Ta113("taillard93", "Ta113.txt", 500, 26371),
+    Ta114("taillard93", "Ta114.txt", 500, 26456),
+    Ta115("taillard93", "Ta115.txt", 500, 26334),
+    Ta116("taillard93", "Ta116.txt", 500, 26469),
+    Ta117("taillard93", "Ta117.txt", 500, 26389),
+    Ta118("taillard93", "Ta118.txt", 500, 26560),
+    Ta119("taillard93", "Ta119.txt", 500, 26005),
+    Ta120("taillard93", "Ta120.txt", 500, 26457),
+    // 240 instances - VFR-small2015
+    VFR10_10_10_Gap("vfr-small2015", "VFR10_10_10_Gap.txt", 10, 1099),
```

Comment:
Ruben does not run this.
Do we want to exclude them? Maybe if they're not present in the data folder, we skip them and just log a warning?

Reply:
Sounds good. It seems better than failing, and we don't need to change the enum list.